pagina-web.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en
+<head>
+    <meta charset=" UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Swipe Mio</title>
+<link rel="stylesheet" href="Styles.css">
+</head>
+
+<body>
+    <div class="fondo">
+        <img  src="Imagenes/Rectangle 1.png" alt="fondo">
+    </div>
+    <section>
+        <nav class="navegation">
+            <img class="logo"  src="/Imagenes/logo-white.svg" alt="Logo Swipe">
```

Comment:
Hazme un favor, para poderlo visualizar en muchos dispositivos (a parte del tuyo) es necesario que le quites el "/" en todos lados en los que accedes a la carpeta `Imagenes`

así:

```suggestion
            <img class="logo"  src="Imagenes/logo-white.svg" alt="Logo Swipe">
```